src/lls_model.jl

```diff
@@ -170,6 +172,9 @@ function LLSModel(
     ucon = ucon,
     nnzh = 0,
     name = name,
+    nnzj = nvar * ncon,
+    lin_nnzj = nvar * ncon,
```

Comment:
This looks a bit weird when C is an operator, but I'm not sure what would be better.

Reply:
Agreed, but I guess it is equally weird to put a zero.
Thanks for the review!